tests/test_end_to_end.py

```diff
@@ -82,6 +82,21 @@ def wait_for_index(lead_id):
     def export(self, client, random_string):
         return client.create_lead_export(random_string)
 
+    @pytest.fixture
+    def webhook(self, client):
+        data = {
+            'url': 'https://fake.io/notes/closeio/',
```

Comment:
can we use some `example.com` domain here? If anyone ever registers the domain `fake.io`, this domain might actually receive requests.

Reply:
Funny thing: in the end to end tests, whenever I put `example.com` (or even `localhost:8000`) I got an exception:

```
E   closeio.exceptions.CloseIOError: ('{"message": "URL not allowed."}', HttpClientError('Client Error 400: https://app.close.io/api/v1/webhook/',), 'url: https://app.close.io/api/v1/webhook/\nbody: {"url": "https://example.com/notes/closeio/", "events": [{"object_type": "activity.note", "action": "created"}]}')
```